InstaBotPrototype/DBMigrator/Migrations/CreateUsersTable.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Data.Common;
+
+namespace DBMigrator
+{
+    [Indexer(201711290745)]
+    class CreateUsersTable : DBMigration
+    {
+        public CreateUsersTable(object factory, object connection) : base(factory as DbProviderFactory, connection as DbConnection)
+        {
+            ApplyCommand.CommandText = "create table dbo.Users (Id integer, Login nvarchar(100), Password nvarchar(100), primary key(Id))";
```

Comment:
Well, this is working approach, and pretty usefull for those who'll be using this migrator. 
however its good to oneliners, when whole migration could be shipped as one line sql statement. 
while sometimes migrations could took several commands . like changing two related tables or so.

Lets start with this approach, however once more complex migrations will emerge, we'll need to refactor both, migrator and all migrations.

Reply:
But why cant we separate sql statements with semicolons?
Like that
"create table dbo.Migrations (Name nvarchar(100), ApplyTime DateTime);
create table dbo.Something (Name nvarchar(100), ApplyTime DateTime);
create table dbo.Else (Name nvarchar(100), ApplyTime DateTime)"